manifests/server.pp

```diff
@@ -16,6 +16,7 @@
     $ping             = false,
     $ping_restart     = false,
     $mtu_discovery    = true,
+    $server_mode      = 'normal',
```

Comment:
Your are right, 'normal' is a bad name. Will change it to a bool? And change name to bridge_mode?

I reuse ifconfig and address to build server-bridge configuration, because doesn't like to much parameter where some are excludes others - and `server-bridge` is for me a mix of `ifconfig_pool` and `address` - do you agree?